generators/client/templates/vue/package.json

```diff
@@ -1,19 +1,100 @@
 {
   "dependencies": {
+    "@openapitools/openapi-generator-cli": "1.0.13-4.3.1",
     "@fortawesome/fontawesome-svg-core": "1.2.32",
     "@fortawesome/free-solid-svg-icons": "5.15.1",
     "@fortawesome/vue-fontawesome": "2.0.2",
+    "vue": "2.6.12",
+    "vue-class-component": "7.2.6",
+    "vue-cookie": "1.1.4",
+    "vue-infinite-loading": "2.4.5",
+    "vue-router": "3.4.9",
     "vue-i18n": "8.22.3",
+    "vue-property-decorator": "9.1.2",
+    "vuelidate": "0.7.6",
+    "vuex": "3.6.0",
+    "vue2-filters": "0.11.1",
     "axios": "0.21.1",
     "bootstrap": "4.5.3",
     "bootswatch": "4.5.3",
-    "bootstrap-vue": "2.21.2"
+    "swagger-ui-dist": "3.25.4",
+    "bootstrap-vue": "2.21.2",
+    "sockjs-client": "1.1.4",
+    "webstomp-client": "1.2.0"
   },
   "devDependencies": {
+    "@types/chai": "4.2.11",
+    "@types/chai-string": "1.4.2",
+    "@types/jest": "26.0.20",
+    "@types/mocha": "7.0.2",
+    "@types/node": "14.14.20",
+    "@types/selenium-webdriver": "4.0.9",
+    "@types/sinon": "9.0.10",
+    "@types/vuelidate": "0.7.13",
+    "@typescript-eslint/eslint-plugin": "4.13.0",
+    "@typescript-eslint/parser": "4.13.0",
+    "@vue/eslint-config-prettier": "6.0.0",
+    "@vue/eslint-config-typescript": "7.0.0",
+    "@vue/cli-plugin-eslint": "4.5.10",
+    "@vue/cli-plugin-typescript": "4.5.10",
+    "@vue/cli-service": "4.5.10",
+    "@vue/test-utils": "1.1.2",
+    "autoprefixer": "10.0.1",
+    "babel-core": "7.0.0-bridge.0",
+    "browser-sync": "2.26.13",
+    "browser-sync-webpack-plugin": "2.3.0",
+    "chai": "4.2.0",
+    "chai-as-promised": "7.1.1",
+    "chai-string": "1.5.0",
+    "copy-webpack-plugin": "6.3.0",
+    "css-loader": "3.5.3",
+    "file-loader": "6.2.0",
+    "fork-ts-checker-webpack-plugin": "4.1.6",
+    "friendly-errors-webpack-plugin": "1.7.0",
+    "html-webpack-plugin": "4.5.0",
+    "jest": "26.6.3",
+    "jest-junit": "12.0.0",
+    "jest-serializer-vue": "2.0.2",
+    "jest-sonar-reporter": "2.0.0",
+    "jest-vue-preprocessor": "1.7.1",
+    "merge-jsons-webpack-plugin": "1.0.21",
+    "mini-css-extract-plugin": "0.9.0",
+    "mocha": "7.1.2",
+    "node-notifier": "9.0.0",
+    "numeral": "2.0.6",
+    "optimize-css-assets-webpack-plugin": "5.0.4",
+    "portfinder": "1.0.28",
+    "postcss": "8.2.4",
+    "postcss-import": "14.0.0",
+    "postcss-loader": "4.1.0",
+    "postcss-url": "10.1.1",
+    "protractor": "5.4.4",
+    "rimraf": "3.0.2",
+    "sass": "1.32.4",
+    "sass-loader": "10.1.1",
+    "sinon": "9.2.3",
+    "terser-webpack-plugin": "4.2.3",
+    "ts-jest": "26.4.4",
+    "ts-loader": "7.0.4",
+    "ts-node": "8.10.1",
+    "tslib": "2.1.0",
+    "tslint-config-prettier": "1.18.0",
+    "tslint-eslint-rules": "5.4.0",
+    "tslint-loader": "3.6.0",
+    "eslint": "7.17.0",
+    "eslint-plugin-prettier": "3.3.1",
+    "eslint-plugin-vue": "7.4.1",
     "typescript": "4.1.3",
+    "url-loader": "4.1.1",
+    "webdriver-manager": "12.1.7",
```

Comment:
This should be `"12.1.8"` to fix an issue [I found recently](https://github.com/jhipster/generator-jhipster/issues/13443#issuecomment-759215624).